fhir-examples-generator/src/main/java/com/ibm/fhir/examples/MinimalDataCreator.java

```diff
@@ -100,12 +99,20 @@ private Builder<?> addData(Builder<?> builder, int choiceIndicator, String refer
         return builder;
     }
 
+    /**
+     * For the minimal data creator, we only count required elements
+     */
     @Override
     protected int getMaxChoiceCount(Class<?> resourceOrElementClass, int maxChoiceCount, int levelsDeep) {
```

Comment:
Is levelsDeep no longer required now?

Reply:
If you don't mind, I think I'll just do C for now since its quick+easy and this code is only used to generate examples

Reply:
Sounds reasonable